src/test/java/com/cgdp/library/controller/AuthorControllerTest.java

```diff
@@ -58,16 +60,17 @@ public void should_create_author() throws Exception {
         resultActions
               .andExpect(status().isCreated())
               .andExpect(jsonPath("$.id", is(id.intValue())))
-              .andExpect(jsonPath("$.name", is(authorName)));
+              .andExpect(jsonPath("$.authorName", is(authorName)));
     }
 
     @Test
     public void should_return_bad_request_when_author_name_is_empty() throws Exception {
         // given
-        String authorName = "";
+        String authorName = "John Doe";
```

Comment:
Test is about having empty name, there are already other tests doing similar stuff, use hardcoded json instead of dto since it will throw exception before we can test anything